src/main/java/uk/gov/justice/digital/delius/transformers/PersonalCircumstanceTransformer.java

```diff
@@ -48,5 +50,14 @@ private static KeyValue circumstanceSubTypeOf(CircumstanceSubType circumstanceSu
                 .build();
     }
 
-
+    private static Boolean activeFlagOf(LocalDate startDate, LocalDate endDate) {
```

Comment:
I would change the contract or rename the method and simply to something like this 


private static Boolean activeFlagOf(uk.gov.justice.digital.delius.jpa.standard.entity.PersonalCircumstance personalCircumstance, LocalDate dateToComapre) {
        return !personalCircumstance.getStartDate().isAfter(dateToComapre) && Optional.ofNullable(personalCircumstance.getEndDate()).map(x -> x.isAfter(dateToComapre)).orElse(true);
    }

Reply:
Have made the requested change and it's looking much cleaner and reusable, thank you.